src/Avalonia.Base/Media/FontManager.cs

```diff
@@ -76,7 +74,18 @@ public FontFamily DefaultFontFamily
         /// <summary>
         ///     Get all system fonts.
         /// </summary>
-        public IFontCollection SystemFonts => _fontCollections[SystemFontsKey];
+        public IFontCollection SystemFonts
+        {
+            get
+            {
+                if (TryGetFontCollection(SystemFontsKey, out var fontCollection))
+                {
+                    return fontCollection;
+                }
+
+                throw new InvalidOperationException("System font collection could not be found.");
```

Comment:
See comments below in `TryGetFontCollection`, but we should always returns a collection for this property even if it's empty.